main.go

```diff
@@ -306,13 +305,16 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// Note(aneesh): We SHOULD fetch the table columns ONLY after the call to EnsureCompatibleExportTable.
```

Comment:
nit: We don't usually put names on notes, as they don't imply any further responsibility. We usually do "NB: note text", NB being short for "nota bene".